src/widgets/password-text.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { classNames } from "../common";
+
+import styles from "./input.css";
+
+const PASSWORD_DOT = "\u2022";
```

Comment:
This unicode char "•" more closely matches the Zeplin mock - but the previous value "●" more closely matches the built-in password field dots. This is easy to switch back if we want.